fairworkflows/fairworkflow.py

```diff
@@ -102,13 +101,12 @@ def _get_relevant_triples(uri, rdf):
         method
         """
         q = """
-        SELECT ?s ?p ?o
+        CONSTRUCT { ?s ?p ?o }
         WHERE {
             ?s ?p ?o .
             # Match all triples that are through an arbitrary-length property path related to the
-            # workflow uri. (a|!a) matches all predicates. Binding to workflow_uri is done when
-            # executing.
-            ?workflow_uri (a|!a)+ ?o .
+            # step uri. (<>|!<>) matches all predicates. Binding to step_uri is done when executing.
```

Comment:
I'm a little unclear about this comment - it's talking about step_uri but the previous and current versions are addressing workflow_uri. What am I misunderstanding here...?

Reply:
This erroneously sneaked in, thanks for spotting! I changed it back to workflow_uri.